plugins/admin/pkg/command/registry/add.go

```diff
@@ -35,24 +35,9 @@ type prcmdFlags struct {
 	Password   string
 }
 
-type Registry struct {
-	Auths Auths `json:"auths"`
-}
-type RegistryCred struct {
-	Username string `json:"Username"`
-	Password string `json:"Password"`
-	Email    string `json:"Email"`
-}
-
-type Auths map[string]RegistryCred
-
-//
-//type Info map[string]Person
-
 var prflags prcmdFlags
 
-// addCmd represents the add command
-
+// NewPrAddCommand represents the add command
 func NewPrAddCommand(p *pkg.AdminParams) *cobra.Command {
```

Comment:
What's the "Pr" stand for in "NewPrAddCommand"?

Reply:
Should be a legacy problem with the meaning of `private registry`. Changed all "pr" part to "registry".